src/main/java/core/basesyntax/DateConvertor.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DateConvertor {
+    public static final DateTimeFormatter DT_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public LocalDate convert(String strDate) {
+        LocalDate localDate = LocalDate.parse(strDate, DT_FORMATTER);
+        return localDate;
```

Comment:
[Don’t create temporary variables if you can directly return value from the method](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)